### ISSUE TITLE
V2运行报错

### ISSUE THREAD

**Issue by nieziqi123** [acheong08/ChatGPT#657]
Traceback (most recent call last):
  File "C:/Users/home/Desktop/setup.py", line 13, in <module>
    asyncio.run(main())
  File "C:\Users\cAke\AppData\Local\Programs\Python\Python38\lib\asyncio\runners.py", line 44, in run
    return loop.run_until_complete(main)
  File "C:\Users\cAke\AppData\Local\Programs\Python\Python38\lib\asyncio\base_events.py", line 616, in run_until_complete
    return future.result()
  File "C:/Users/home/Desktop/setup.py", line 4, in main
    chatbot = Chatbot(email=***)
  File "C:\Users\cAke\AppData\Local\Programs\Python\Python38\lib\site-packages\revChatGPT\V2.py", line 120, in __init__
    self.login(email, password)
  File "C:\Users\cAke\AppData\Local\Programs\Python\Python38\lib\site-packages\revChatGPT\V2.py", line 172, in login
    auth.begin()
  File "C:\Users\cAke\AppData\Local\Programs\Python\Python38\lib\site-packages\OpenAIAuth\OpenAIAuth.py", line 82, in begin
    self.__part_two()
  File "C:\Users\cAke\AppData\Local\Programs\Python\Python38\lib\site-packages\OpenAIAuth\OpenAIAuth.py", line 113, in __part_two
    self.__part_three(token=csrf_token)
  File "C:\Users\cAke\AppData\Local\Programs\Python\Python38\lib\site-packages\OpenAIAuth\OpenAIAuth.py", line 155, in __part_three
    self.__part_four(url=url)
  File "C:\Users\cAke\AppData\Local\Programs\Python\Python38\lib\site-packages\OpenAIAuth\OpenAIAuth.py", line 180, in __part_four
    response = self.session.get(
  File "C:\Users\cAke\AppData\Local\Programs\Python\Python38\lib\site-packages\tls_client\sessions.py", line 363, in get
    return self.execute_request(method="GET", url=url, **kwargs)
  File "C:\Users\cAke\AppData\Local\Programs\Python\Python38\lib\site-packages\tls_client\sessions.py", line 346, in execute_request
    raise TLSClientExeption(response_object["body"])
tls_client.exceptions.TLSClientExeption: failed to do request: Get "https://auth0.openai.com/authorize?client_id=TdJIcbe16WoTHtN95nyywh5E4yOo6ItG&scope=openid%20email%20profile%20offline_access%20model.request%20model.read%20organization.read&response_type=code&redirect_uri=https%3A%2F%2Fexplorer.api.openai.com%2Fapi%2Fauth%2Fcallback%2Fauth0&audience=https%3A%2F%2Fapi.openai.com%2Fv1&prompt=login&state=qU7DVNFUUWgtBO97UzDJuxNsmfZOkIXubdOmQSggikU&code_challenge=80i-xfKUYRC00xd1kRrubtI87oE7ZTy0B9-JqrHnVT0&code_challenge_method=S256": dial tcp 104.18.13.135:443: i/o timeout (Client.Timeout exceeded while awaiting headers)

看上去无法访问登录页面

**Comment by acheong08** [comment]
Duplicate of https://github.com/acheong08/ChatGPT/issues/652

**Comment by acheong08** [comment]
TL;DR: 中国无法用

**Comment by acheong08** [comment]
正在写
